Formalisation(shacl)/Core/PiecesShape/Dataset.ttl

```diff
@@ -153,3 +154,26 @@
   ];
 
 sh:targetClass foaf:Agent .
+
+
+:KindShape a sh:NodeShape;
+  sh:property     
+  [ 
+    sh:path vcard:hasEmail ;
+    sh:nodeKind sh:IRI ;
```

Comment:
As discussed with Patrick, maybe we can add a `sh:pattern "mailto:\w+@\w+\.\w+"` here.

Reply:
done